CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.0
+* The version of master branch will be start with 0.1.x, and Revit2020 branch will still be 0.0.xx.
+* LibG should be more version compatible.
```

Comment:
What do you mean by this statement exactly?

Reply:
The version for master branch will be like 0.1.x, and RC2.1.0_Revit2020 will be 0.0.x, and in the future,  after a new major branch is split, that branch will have version 0.1.x, and the master will be 0.2.x.
I add the statement just to help others to understand how we define version number.